templates/sway-predicate-test-rs/template/Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["{{authors}}"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-fuels = { version = "0.66", features = ["fuel-core-lib"] }
+fuels = { version = "0.66.1", features = [] }
```

Comment:
```suggestion
fuels = "0.66.1"
```

Reply:
are we able to do this or is there a reason we want an empty features array?

Reply:
Done.